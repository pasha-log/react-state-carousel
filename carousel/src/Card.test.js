import React from 'react';
import { render } from '@testing-library/react';
import Card from './Card';

it('should render', () => {
	render(<Card caption="This is a fake picture" src="picture.png" currNum={1} totalNum={1} />);
});

it('should match snapshot', () => {
	const { asFragment } = render(<Card caption="This is a fake picture" src="picture.png" currNum={1} totalNum={1} />);
	expect(asFragment()).toMatchSnapshot();
});
