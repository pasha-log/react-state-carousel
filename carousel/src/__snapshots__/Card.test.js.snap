// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      This is a fake picture
    </h4>
    <img
      alt="This is a fake picture"
      class="Card-image"
      src="picture.png"
    />
    <small
      class="Card-small"
    >
      Image 1 of 1.
    </small>
  </div>
</DocumentFragment>
`;
